lib/takeaway.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'menu'
+class Takeaway
+
+  attr_reader :menu_klass, :total_dishes, :order, :total_cost
+
+  def initialize(menu_klass = Menu.new)
+    @menu = menu_klass
+    @order = Hash.new
```

Comment:
Style/EmptyLiteral: Use hash literal `{}` instead of `Hash.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)